python/turbine_models/gen_external_params/gen_external_params.py

```diff
@@ -72,35 +72,52 @@ def forward(self, x):
         all_weights.update(int_weights)
     return all_weights
 
+def gen_external_params(hf_model_name:str = "meta-llama/Llama-2-7b-chat-hf",
```

Comment:
might get kind of annoying to change the default from the "meta-llama/Llama-2-7b-chat-hf" model for us, but if we do switch it should probably be  "Trelis/Llama-2-7b-chat-hf-function-calling-v2"

Reply:
Gotcha, on my todo list for the next push.